package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "@google-cloud/common": "^0.12.0",
     "continuation-local-storage": "^3.1.4",
+    "extend": "^3.0.0",
```

Comment:
ah, I see 💡 
I looked at https://github.com/sindresorhus/deep-assign thinking it would use Object.assing underneath but does not appear so, `extend` is fine then (although `deep-assign` seems less bloated)